test/new.jl

```diff
@@ -2417,7 +2417,7 @@ end
             Pkg.add(Pkg.PackageSpec(uuid=exuuid, version=v"0.5.3"))
         catch e
             @test e isa ResolverError
-            @test occursin("possible versions are: 0.5.1 or uninstalled", e.msg)
+            @test occursin(r"possible versions are: .*0.5.1.* or uninstalled", e.msg)
```

Comment:
```suggestion
            @test occursin(r"possible versions are: .*\b0.5.1\b.* or uninstalled", e.msg)
```
I wonder if there's a way to make this a bit more precise? Maybe something like this, requiring word boundaries?

Reply:
Could be even more precise and workout the right regex for color escape sequences.
but the additional difficulty reading might offset the surity added to the tests
the  `\b` might strike the right balance

Reply:
That was my thinking as well. Just make sure that `0.5.1` is in there as a word.

Reply:
Also should be `0\.5\.1`.